models/featureFlags.js

```diff
@@ -0,0 +1,137 @@
+const firestore = require('../utils/firestore')
+const featureFlagModel = firestore.collection('featureFlags')
+const userModel = require('./users')
+
+/**
+ * Fetch all tasks
+ *
+ * @return {Promise<featureFlags|Array>}
+ */
+const fetchFeatureFlag = async () => {
+  try {
+    const snapshot = await featureFlagModel.get()
+    const featureFlags = []
+    snapshot.forEach((doc) => {
+      featureFlags.push({
+        id: doc.id,
+        ...doc.data()
+      })
+    })
+    const users = []
+    const result = {}
+
+    featureFlags.forEach((item) => {
+      if (!users.includes(item.owner)) { users.push(item.owner) }
+    })
+
+    const getImage = async (usersData) => {
+      return await userModel.fetchUserImage(usersData)
+    }
+
+    let start = 0
+    let end = 10
+
+    for (let i = 0; i < Math.ceil(users.length / 10); i++) {
+      const usersData = users.slice(start, end)
```

Comment:
You can also do the logic in loop using just `i`.

Don't need `start` and `end` vars.